### ISSUE TITLE
stopWords and pre-existing data does not work in databricks

### ISSUE THREAD

**Issue by alexandrahitson** [zinggAI/zingg#718]
**Describe the bug**
1. There is little to no difference in match results before and after loading and setting a custom list of stopWords ([stopwords.csv](https://github.com/zinggAI/zingg/files/13298781/stopwords.csv)).
2. I am getting a zingg Client exception when attempting to train zingg on a pre-trained dataset.

**To Reproduce**

_Stopwords:_
1. Databricks notebook was constructed based on zingg image databricks example [here](https://notebooks.databricks.com/notebooks/RCG/product-er-with-images/product-er-with-images_1.html).
2. Import correct libraries
3. Run scripts to check the existence of .jar file and .whl file.
4. setStopWords on the field for which I want stopwords
```
#set field definitions 
#please change these 
field_1 = FieldDefinition("field_1", "string", MatchType.DONT_USE)
field_2 = FieldDefinition("field_2", "string", MatchType.FUZZY)
field_2.setStopWords("../stopwords.csv")
field_3 = FieldDefinition("field_3", "string", MatchType.DONT_USE)

fieldDefs = [field_1, field_2, field_3]
args.setFieldDefinition(fieldDefs)
```
5. findTrainingData and label phases
6. Initialize trainMatch phase and run
7. Look at and save results

_Pre-trained Data:_
1. Steps 1-3 from above
2. Label a few records from the input data source and save the marked and unmarked records
3. Set the training samples in args (pre-trained data set):
```
trainingSamples_schema = "z_cluster string, z_isMatch integer, field_1 string, field_2 string, field_3 string"
trainingSamples = Pipe("trainingSamples", "csv")
trainingSamples.addProperty("location", "location/file.csv")
trainingSamples.addProperty("delimiter", ",")
trainingSamples.addProperty("header", "true")
trainingSamples.addProperty("schema", trainingSamples_schema)
args.setTrainingSamples(trainingSamples)
```

8. Initialize the train function and execute it.
9. Exception results included in screenshot section

**Expected behavior**
I expect the addition of stop words on my field of choice to provide more than a very small difference in matching results given the nature of my dataset.
I expect the training phase to run on the pre-trained dataset in order to train a model based on those groupings and matches.

**Screenshots**
![Screenshot from 2023-11-08 10-49-49](https://github.com/zinggAI/zingg/assets/140414696/0e20071d-9d20-4301-9c2f-c128ba071148)

![Screenshot from 2023-11-08 10-50-39](https://github.com/zinggAI/zingg/assets/140414696/a23ebee4-52f8-4f60-bbd2-065796ed4c06)

![Screenshot from 2023-11-08 10-51-23](https://github.com/zinggAI/zingg/assets/140414696/882c6c99-a003-44bf-906d-3d6d481aa968)

![Screenshot from 2023-11-08 10-51-41](https://github.com/zinggAI/zingg/assets/140414696/d7b22ecc-f33d-4610-9520-6f20f7d6a79d)

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser : Chrome
 - Version : Version 119.0.6045.105 (Official Build) (64-bit)

**Additional context**
The pre-trained data set I am loading was manually labeled by me. Baseline, the model words, but I am trying to make fine tuning improvements via stop words and my own pre-trained/pre-labeled dataset.

**Comment by sonalgoyal** [comment]
Thanks for reporting @alexandrahitson, will try and schedule this as soon as possible

**Comment by sonalgoyal** [comment]
@gnanaprakash-ravi can oyu try this locally using the zingg python api?

**Comment by gnanaprakash-ravi** [comment]
Hi, 
setStopWords is working fine for my testing using Zingg Python API
![image](https://github.com/zinggAI/zingg/assets/89972506/063b4c43-2149-4ec0-a8c4-67938f93d393)
I set up some stopwords in stopwords.csv
![image](https://github.com/zinggAI/zingg/assets/89972506/d6131211-e61b-47ff-9614-3791f9b6c8ae)
and made the corresponding changes in the test.csv file
Result:
![image](https://github.com/zinggAI/zingg/assets/89972506/5a4c4874-0490-4788-9585-d2bee51e864b)

**Comment by vikasgupta78** [comment]
no issue found in stop words and training sample issue fixed as part of issue #743 , will be available in 0.4.0, to be released soon.

@alexandrahitson
